lib/utils.js

```diff
@@ -9,11 +9,10 @@
  * Module dependencies.
  */
 
-var debug = require('debug')('mocha:watch');
 var fs = require('fs');
-var glob = require('glob');
 var path = require('path');
-var join = path.join;
+var debug = require('debug')('mocha:watch');
```

Comment:
maybe just `mocha:utils` in case we want to use it elsewhere in this file.  again, realize you didn't do this

Reply:
I moved it to `exports.watch`, where it was being used.
We can add another module variable if the need arises.